### PR TITLE
VSL-165: Added nft safe overview

### PR DESCRIPTION
**Pull request by linxiao-li**
## Description
1. Add NFTs section in safe overview
2. Renamed nav tab "assets" to "NFTs" according to the design

## Demo / Test Result
- When treasury doesn't have any collection
Screen Shot 2022-09-20 at 9 01 49 AM
- When treasury has some collections and user click view all link

https://user-images.githubusercontent.com/7423845/191308340-25c73d05-2bb7-4572-843c-256829728e9e.mov

### CODE REVIEW

File: packages/client/src/components/SafeOverview.js [hunk 1]
```diff
@@ -59,34 +65,81 @@ function SafeOverview({ allBalance }) {
...
+          )}
+          {!!collectionsForDisplay.length && (
+            <div className="columns mt-1 is-multiline is-variable is-1">
+              {collectionsForDisplay.map(({ contractName, count }) => (
+                <div className="column is-half py-1" key={contractName}>
+                  <p className="has-background-tertiary-purple rounded-sm p-3 is-flex is-flex-direction-row is-justify-content-space-between">
+                    <span className="flex-3 has-text-truncate">
+                      {contractName}
+                    </span>
+                    <span className="flex-1 has-text-right">
```
Comment (KsenijaZivkovic) [inline packages/client/src/components/SafeOverview.js:123]: Add is-family-monospace to className

File: packages/client/src/components/SafeOverview.js [hunk 2]
```diff
@@ -59,34 +65,81 @@ function SafeOverview({ allBalance }) {
...
+              </div>
+            </div>
+          )}
+          {!!collectionsForDisplay.length && (
+            <div className="columns mt-1 is-multiline is-variable is-1">
+              {collectionsForDisplay.map(({ contractName, count }) => (
+                <div className="column is-half py-1" key={contractName}>
+                  <p className="has-background-tertiary-purple rounded-sm p-3 is-flex is-flex-direction-row is-justify-content-space-between">
+                    <span className="flex-3 has-text-truncate">
+                      {contractName}
```
Comment (KsenijaZivkovic) [inline packages/client/src/components/SafeOverview.js:121]: Name can be extracted from MetadataViews getNFTCollectionDisplay.
All NFT contracts should have it defined, so we need a script to fetch this.
Maybe even an icon can be extracted this way
```
return MetadataViews.NFTCollectionDisplay(
                        name: "Zeedz",
                        description: "Zeedz is the first play-for-purpose game where players reduce global carbon emissions by collecting plant-inspired creatures: Zeedles. They live as NFTs on an eco-friendly blockchain (Flow) and grow with the real-world weather.",
                        externalURL: MetadataViews.ExternalURL("https://play.zeedz.io"),
                        squareImage: mediaSquare,
                        bannerImage: mediaBanner,
                        socials: socials
                        )
```

File: packages/client/src/components/SafeOverview.js (hunk 1 above)
Comment (linxiao-li) [reply packages/client/src/components/SafeOverview.js:123]: Good catch! Thank you!

File: packages/client/src/components/SafeOverview.js (hunk 2 above)
Comment (linxiao-li) [reply packages/client/src/components/SafeOverview.js:121]: Do you mean using a async call to get the NFTCollectionDisplay?

Comment (KsenijaZivkovic) [reply packages/client/src/components/SafeOverview.js:121]: Yup!

Comment (linxiao-li) [reply packages/client/src/components/SafeOverview.js:121]: ticket created here https://linear.app/dappercollectives/issue/VSL-217

File: packages/client/src/components/SafeOverview.js [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
+  const emptyNftBlocks = new Array(4 - collectionsForDisplay.length).fill(0);
   return (
     <>
       <div className="column p-0 mt-5 is-flex is-full">
         <h2>Safe Overview</h2>
       </div>
       <div className="column p-0 mt-4 mb-5 is-flex is-full">
         <div
-          className={balanceClasses.join(" ")}
+          className="is-flex is-flex-direction-column flex-1 rounded-sm has-background-primary-purple has-shadow has-text-white p-5 mr-6"
```
[the remark this answers is not in the document]
Comment (linxiao-li) [reply packages/client/src/components/SafeOverview.js:33]: Done! Removed them on the token side, for nft view we do columns

**Review by KsenijaZivkovic — APPROVED** [review]

**Review by amyliu6 — APPROVED** [review]

**Review by jacksonConrad — APPROVED** [review]

**Pull request merged**